### PR TITLE
Added FixedZone for missing time zone file

### PR DESCRIPTION
**Pull request by MichaelS11**

### CODE REVIEW

**Comment by MichaelS11** [comment]
@mattn Could you please review and merge?

File: globals.go [hunk 1]
```diff
@@ -182,7 +183,13 @@ func init() {
 	for i := 0; i < len(timeLocations); i++ {
 		timeLocations[i], err = time.LoadLocation(timeLocationNames[i])
 		if err != nil {
-			panic(err)
+			name := "GMT"
+			if i < 12 {
+				name += strconv.FormatInt(int64(i-12), 10)
+			} else {
```
Comment (mattn) [inline globals.go]:
```go
} else if i > 12 {
```

**Comment by MichaelS11** [comment]
@mattn Updated

**Comment by mattn** [comment]
Thank you!

**Pull request merged**